docfx/examples/IceRpc.RequestContext.Examples/RequestContextInterceptorExamples.cs

```diff
@@ -19,18 +20,29 @@ public static async Task UseRequestContext()
             .UseRequestContext()
             .Into(connection);
 
-        var greeter = new GreeterProxy(pipeline);
-
         // Create a feature collection holding an IRequestContextFeature.
         IFeatureCollection features = new FeatureCollection().With<IRequestContextFeature>(
             new RequestContextFeature
             {
                 ["UserId"] = Environment.UserName.ToLowerInvariant(),
                 ["MachineName"] = Environment.MachineName
             });
+        #endregion
```

Comment:
I'd rename this region to `CreateRequestContext`

Reply:
This is named after the `UseRequestContext()` method, which is what the example is about.